pkg/datasource/compat/wrapper.go

```diff
@@ -358,6 +379,10 @@ func (ev *EventWrapper) SetPodMetadata(container types.Container) {
 		if ev.ownerNameAccessor.IsRequested() {
 			ev.ownerNameAccessor.PutString(ev.Data, container.K8sOwnerReference().Name)
 		}
+		if ev.podLabelsAccessor.IsRequested() {
+			podLabels, _ := json.Marshal(k8s.PodLabels)
```

Comment:
I am thinking if it will make sense combine the labels as string till we support https://github.com/inspektor-gadget/inspektor-gadget/issues/3032 @alban / @mauriciovasquezbernal thoughts?

Reply:
I would suggest the same. For the time I would keep it the same as the `kubectl ... --show-labels` output:

```
➭ kubectl get all -A --show-labels
NAMESPACE     NAME              READY   STATUS   RESTARTS    AGE    LABELS
gadget        pod/gadget-qbfhr  1/1     Running  0           6h2m   controller-revision-hash=5b9ccb6798,k8s-app=gadget,pod-template-generation=1
```

So it would be `key1=value1,key2=value2`